R/slide.R

```diff
@@ -225,6 +248,67 @@ epi_slide <- function(x, f, ..., before, after, ref_time_values,
     after <- 0L
   }
 
+  if (autocomplete_windows) {
+    assert_list(fill, names = "unique")
+    fill_vars_not_in_x <- !(names(fill) %in% colnames(x))
+    if (any(fill_vars_not_in_x)) {
+      Warn(
+        c(
+          "Some names provided in `fill` do not correspond to column names in
+           the input data. Any columns not specified in `fill` will be
+           completed with missing values (`NA`) by default."
```

Comment:
Second sentence seems a little tangential.  Might want to focus on what they need to do to fix it.  (Does tidyr not abort when there's this sort of typo?)

Reply:
`tidyr::complete` silently ignores list items that don't correspond to observed column names.